### PR TITLE
addedAt field from chrome & firefox should be set for RawKeep.createdDat...

### PR DESCRIPTION
**Pull request by ahsu1230**
...e

@andrewconner 
From my understanding, the JSON the Extension is already sending a json with field `addedAt`. We simply look for that field in `RawKeepFactory.toRawKeep`, set it for `createdDate`. `createdDate` gets set for `RawBookmarkRepresentation.keptAt` in `RawKeepImporterPlugin.processBatch`, which eventually translates to `Keep.keptAt`

### CODE REVIEW

**Comment by ahsu1230** [comment]
@stkem

**Comment by stkem** [comment]
lgtm

File: kifi-backend/modules/shoebox/app/com/keepit/model/RawKeep.scala [hunk 1]
```diff
@@ -46,9 +46,10 @@ class RawKeepFactory @Inject() (airbrake: AirbrakeNotifier) {
     val title = (json \ "title").asOpt[String]
     val url = (json \ "url").asOpt[String].getOrElse(throw new Exception(s"json $json did not have a url"))
     val isPrivate = (json \ "isPrivate").asOpt[Boolean].getOrElse(true)
+    val addedAt = (json \ "addedAt").asOpt[DateTime]
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/model/RawKeep.scala:49]: Does Firefox use "ADD_DATE" instead? (re: your asana post)

Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/model/RawKeep.scala:49]: chrome and firefox both use ADD_DATE in the <a> tag for every exported link

Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/model/RawKeep.scala:49]: This code looks correct to me. The browser extension sends `"addedAt"`.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/model/RawKeep.scala:49]: Great

**Pull request merged**